src/constants.ts

```diff
@@ -96,8 +96,9 @@ export const addresses: IAddresses = {
     BONDINGCALC_ADDRESS: "0xcaaa6a2d4b26067a391e7b7d65c16bb2d5fa571a",
     CIRCULATING_SUPPLY_ADDRESS: "0x0efff9199aa1ac3c3e34e957567c1be8bf295034",
     TREASURY_ADDRESS: "0x31f8cc382c9898b273eff4e0b7626a6987c846e8",
-    // TODO (appleseed-lusd): swap this out
```

Comment:
all these existing arbitrum contract addresses were added previously. I think they're not valid & could be removed?